cmd/cmdutils.go

```diff
@@ -170,3 +173,48 @@ func printUnmountedStorage(client *occlient.Client, applicationName string) {
 	}
 	fmt.Println("")
 }
+
+// GetAppName returns application name
+func getAppName(client *occlient.Client, projectName string) string {
+
+	if Application != "" {
+
+		_, err := application.Exists(client, Application, projectName)
+		if err != nil {
+			checkError(err, "")
+		}
+		return Application
+	}
+	applicationName, err := application.GetCurrent(projectName)
+	checkError(err, "unable to get current application")
+
+	return applicationName
+}
+
+// GetProjectName returns project name
+func getProjectName(client *occlient.Client) string {
+	// Namespace is `--project` flag
+	if Namespace != "" {
```

Comment:
using global variable :-( it might be better to pass this as a parameter rather than using it as global variable